versioned_docs/version-2.0/how_to_guides/index.md

```diff
@@ -118,9 +118,9 @@ Step-by-step guides that cover key tasks and operations in LangSmith.
 - [ ] Group traces as threads
 - [ ] View threads
 
-## Prompt hub
+## Prompts
 
-- [ ] Create a prompt
+- [Create a prompt](./how_to_guides/prompts/create_a_prompt)
```

Comment:
All H2 should be nested under this bullet (see examples in tracing / evaluations section)